src/Client/Client.php

```diff
@@ -30,8 +31,14 @@ public function request(RequestInterface $request): ResponseInterface
 
             $response = $this->client->send($psrRequest);
 
+            $contents = $response->getBody()->getContents();
+
             $this->logger->logResponse($request->getEndpoint(), $psrRequest, $response);
 
+            if (str_contains($contents, 'errors')) {
```

Comment:
Is it safe to check for errors like this? API does not return error status?

Reply:
Unfortunatelly not :/ Always 200.